text_extensions_for_pandas/spanner/extract.py

```diff
@@ -52,17 +53,42 @@ def load_dict(file_name: str, tokenizer: "spacy.tokenizer.Tokenizer"):
 
     :param tokenizer: Preconfigured tokenizer object for tokenizing
     dictionary entries.  **Must be the same configuration as the tokenizer
-    used on the target text!**
+    used on the target text!**  If None, this method will use SpaCy's default
+    English tokenizer.
 
-    :return: a `pd.DataFrame` with the normalized entries.
+    :return: a `pd.DataFrame` with the normalized, tokenized dictionary entries.
     """
     with open(file_name, "r") as f:
         lines = [
-            line.strip() for line in f.readlines() if len(line) > 0 and line[0] != "#"
+            line.strip() for line in f.readlines()
+            if len(line.strip()) > 0 and line[0] != "#"
         ]
 
+    return create_dict(lines, tokenizer)
+
+
+def create_dict(entries: Iterable[str],
+                tokenizer: "spacy.tokenizer.Tokenizer" = None) -> pd.DataFrame:
+    """
+    Create a dictionary from a list of entries, where each entry is expressed as a
+    single string.
+
+    Tokenizes and normalizes the dictionary entries.
+
+    :param entries: Iterable of strings, one string per dictionary entry.
+
+    :param tokenizer: Preconfigured tokenizer object for tokenizing
+    dictionary entries.  **Must always tokenizer the same way as the tokenizer
```

Comment:
i think the grammer is off. Do you mean "Must always tokenize the same way as the tokenizer..."?

Reply:
Oops! Fixed now.